### PR TITLE
Video and Ads packs

### PR DESCRIPTION
**Pull request by asllop**
## Summary

Added the following packs:

- iOS / tvOS video agent
- Android video agent
- Web video agent, with the following video trackers:
  - JWPlayer tracker
  - HTML5 player tracker
  - ThePlatform player tracker
  - VideoJS player tracker
  - Akamai player tracker
  - Chromecast tracker (not a web player, but dependant on the web video agent because is JS)
- Also depending on the web video agent, the following Ads trackers:
  - Google Publisher Tag (GPT)
  - Prebid

### CODE REVIEW

File: packs/ads/web/ads-web-gpt/config.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: ads-web-gpt
+title: Agent for Google Publisher Tags
+description: Agent to monitor web applications using GPT.
+summary: Agent to monitor web applications using GPT.
+level: Community
+authors:
+  - New Relic
```
Comment (polfliet) [inline packs/ads/web/ads-web-gpt/config.yml:7]: We have been using New Relic Labs for contributions from NR Labs. Can you update the author?

**Review by polfliet — CHANGES_REQUESTED** [review]
@jbeveland27 maybe a stupid question but do we support the directory structure as used here packs/ads/web/ads-web-gpt/? We have been using 2 levels deep, but this is 3 and I just want to confirm the API will find those

Thanks @asllop great to have these represented in New Relic I/O!

File: packs/ads/web/ads-web-gpt/config.yml (hunk 1 above)
Comment (asllop) [reply packs/ads/web/ads-web-gpt/config.yml]: Done!

**Comment by jbeveland27** [comment]
> @jbeveland27 maybe a stupid question but do we support the directory structure as used here packs/ads/web/ads-web-gpt/? We have been using 2 levels deep, but this is 3 and I just want to confirm the API will find those

@polfliet Yep, that'll work. [Here's an example](https://regex101.com/r/OIxMfj/3) of the regex we use. As long as it's in the `packs` directory and there's a `config.yml|yaml` at the end, we'll match

**Review by polfliet — APPROVED** [review]

**Comment by polfliet** [comment]
> > @jbeveland27 maybe a stupid question but do we support the directory structure as used here packs/ads/web/ads-web-gpt/? We have been using 2 levels deep, but this is 3 and I just want to confirm the API will find those
> 
> @polfliet Yep, that'll work. [Here's an example](https://regex101.com/r/OIxMfj/3) of the regex we use. As long as it's in the `packs` directory and there's a `config.yml|yaml` at the end, we'll match

thanks for confirming!

**Pull request merged**